server/github_token_handler_test.go

```diff
@@ -0,0 +1,67 @@
+package server
+
+import (
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+
+	"github.com/mattermost/release-bot/model"
+	"github.com/mattermost/release-bot/store"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGithubTokenHandlerInvalidRequests(t *testing.T) {
+	eventContextStore := store.NewEventContextStore()
+	eventContextStore.Store(createWorkflowRunEvent(t), "12345")
+
+	handler := newGithubTokenHandler(&mockClientCache{}, eventContextStore)
+	t.Run("Method Not Allowed", func(t *testing.T) {
```

Comment:
this is a great case for table driven as we discussed last time as there is a lot of repeatability here. no need to change anything.

Reply:
Converted to table driven tests.